v2/README.md

```diff
@@ -0,0 +1,42 @@
+# Gengo: a framework for building simple code generators
```

Comment:
Can we get an OWNERS file added here in the v2 directory?

Reply:
I assumed root was OK, but can copy that over if you want to make it different?  But do we want it to be different?  I don't think so.